frontend/src/components/postById/PostById.cy.js

```diff
@@ -0,0 +1,30 @@
+import PostById from './PostById';
+import Post from "../post/Post";
+const navigate = () => {};
+
+
+describe("PostById", () => {
+    it('calls the /posts/:postId endpoint and displays the post', () => {
+        const {postId} = 1;
```

Comment:
I don't think this line is needed as the Integer 1 is not an object, so object destructing would not work in this case.
A valid destructing would look like this `const {postId} = {postId: 1};`

Reply:
When changing to` const {postId} = {postId: 1}`, our test fails so I've kept it as `const {postId} = 1`

Reply:
Yeah this is wired. I will take a look after lunch.

Reply:
I think changing to `const postId = 1;`
And then change the intercept to `cy.intercept("GET", `/posts/**`, (req) => {`
That would make more sense and pass the tests.